web/crux/src/app/image/image.service.ts

```diff
@@ -104,8 +106,15 @@ export default class ImageService {
 
     this.imagesAddedToVersionEvent.next(images)
 
+    const registryLookup = images.reduce((prev, current) => {
```

Comment:
we dont need a lookup. the registry is there in the image

Reply:
oh 🙈